### DISCUSSION TITLE
Suggestion: Keep bug reports in english

### DISCUSSION THREAD

**Discussion by RaptorX** [thqby/vscode-autohotkey2-lsp#639] [Ideas]
I am glad @PythonYunfei is reporting a lot of bugs but he is always doing it in your native language.

I know that you guys find it easier to work that way but other people may want to keep track of the bugs that have been fixed. I for one find it very hard to have to translate every single one of the emails i get, to understand what is going on.

Usually is best to keep the bug reports in english because that way other people can chime in and provide examples of whether the bug is affecting us or provide additional information. But not knowing your language makes that pretty hard.

I keep track of the bugs because I have a channel in which I keep people updated of new features/bug fixes related to autohotkey and, as v2 heavily depends on your extension, it makes it also hard to be able to report on those to people who are learning about the tool.

Would you consider keeping the bug request forum in english?

EDIT: mispellings

**Comment by marium0505** [comment]
I agree completely. I’ve actually been thinking about making a similar issue myself. 

Issues that are made in other languages also make it hard for people to search for existing issues to ensure they don’t make a duplicate issues, as you would have to search for words/phrases in all the languages that are used in issues.

**Reply by RaptorX** [reply]
I hadn't thought of that. Which is a way more important point. 

How can I know if the issue I'll report isn't a duplicate in another language?

**Comment by PythonYunfei** [comment]
I'm sorry about this my friend🤣, although it wasn't intentional. 

My English is not that good. Sometimes I have to take time to figure out how to express the situation in a clear way, even in my native language.  If I write it in English, it will take even much more time.  Also, I must use "Dict" to translate.  To be worse, even if I have written it in my less better English, there is no guarantee that you could understand it well instead, as well as @thqby of course. 

There are many people using this extension in my country too, where some's English is not so good like me and even some cannot speak English.  So at least, in any case, reporting to fix, is more or less better than nothing happening, I think. Whether it happens, is more important than how it happens, as really well fixing it should be always more important than reading.

However, no matter what, I will express myself in English as much as possible as l could.

**Reply by RaptorX** [reply]
I totally get it, but imagine if I used my native language (Spanish) to report bugs, while you used yours. It would be a nightmare to keep track of who reported what. Hehe. 

Also Google does a good translation when i go to GitHub, but I'm mainly referring to the emails we get. I can't translate those easily and that's what I really follow, I only come to github when the issue concerns me or is interesting enough or I might have an idea of what's going on. 

Have you played with chat gpt? It's really damn good at translating. You can type your message to gpt in your language, tell it to translate it and then copy paste. That means only you have to translate. The way you are doing it now every person who doesn't speak your language (which is a lot of people) has to translate it :)

Thanks for understanding!

**Reply by marium0505** [reply]
Practise makes perfect. English isn't my native language either (Norwegian is). :) 

I also support @RaptorX suggestion about ChatGPT.

**Comment by PythonYunfei** [comment]
If @thqby  reply me in English as well, I will try to post in English too.

> as really well fixing it should be always more important than reading.

Keeping above this in mind, perhaps I would l sometimes try two language versions together which contains English, for my post.  The native language is for @thqby who is aiming to fix, while English is for you and others to read.

**Comment by thqby** [comment]
If the translation tools I can use at present are accurate enough, I can accept any language to submit comments, otherwise it will cause obstacles to communication. Of course, using the same language can make it easier for more people to communicate.
